### PR TITLE
adding an FAQ to mySQL data types

### PR DESCRIPTION
**Pull request by alex-emerich**

### CODE REVIEW

File: content/05-mysql/07-introduction-to-data-types.mdx [hunk 1]
```diff
@@ -302,3 +302,61 @@ If you are using [Prisma Client](https://www.prisma.io/docs/reference/tools-and-
...
+The declaration syntax for a `DECIMAL` column is `DECIMAL(M, D). The ranges of values for the argumments are as follows:
+
+* *M* is the maximum number of digits (the precision). It has a range of 1 to 65.
+* *D* is the number of digits to the right of the decimal point (the scale). It has a range of 0 to 30 and must be no larger than *M*. 
+
+</details>
+
+<details><summary>How does MySQL store `TEXT`?</summary>
+
+The [storage requirements](https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html) for string types in MySQL can be represented in the following table with *M* representing the declared column length in characters and *L* representing the actual length in bytes of a given string value.
```
Comment (imchairmanm) [inline content/05-mysql/07-introduction-to-data-types.mdx:319]: Was curious about the inclusion of the "*M*" variable here as that doesn't appear to be in the table.

File: content/05-mysql/07-introduction-to-data-types.mdx [hunk 2]
```diff
@@ -302,3 +302,61 @@ If you are using [Prisma Client](https://www.prisma.io/docs/reference/tools-and-
...
+
+```
+VARCHAR(10)
+``` 
+
+</details>
+
+<details><summary>What is the maximum length of `VARCHAR` in MySQL?</summary>
+
+Values in `VARCHAR` columns are variable-length strings. The length can be specified as a value from 0 to 65,535.
```
Comment (imchairmanm) [inline content/05-mysql/07-introduction-to-data-types.mdx:346]:
```suggestion
Values in `VARCHAR` columns are variable-length strings. The maximum length can be specified as a value from 0 to 65,535.
```

**Review by imchairmanm — APPROVED** [review]
Looks good :+1: Just a couple of small things.

File: content/05-mysql/07-introduction-to-data-types.mdx (hunk 1 above)
Comment (alex-emerich) [reply content/05-mysql/07-introduction-to-data-types.mdx:319]: Ah good catch. M would be included for binary or char. Doesn't need to be there for TEXT

**Pull request merged**